src/clusterfuzz/_internal/metrics/logs.py

```diff
@@ -32,6 +33,29 @@
 _default_extras = {}
 
 
+@functools.cache
+def _current_source_version():
+  """Return the current source version:
+    - Arbitrary value local runs
+    - from base.utils.current_source_version():
+      - SOURCE_VERSION_OVERRIDE if set
+      - from the package's clusterfuzz-source.manifest if present
+      - None"""
+  # local imports to avoid circular imports
+  # (base.memoize > base.persistent_cache > base.utils > base.memoize)
+  from clusterfuzz._internal.base.utils import current_source_version
+  from clusterfuzz._internal.base.utils import utcnow
+  if os.getenv('PY_UNITTESTS'):
+    timestamp = utcnow().strftime('%Y%m%d%H%M%S-utc')
+    components = [timestamp, "PY_UNITTESTS"]
+    return '-'.join(components)
+  if _is_local():
+    timestamp = utcnow().strftime('%Y%m%d%H%M%S-utc')
+    components = [timestamp, os.environ['USER'], "LOCAL"]
+    return '-'.join(components)
```

Comment:
I think we already have the timestamp in the logs.

Reply:
yeah I was keeping the format of current_source_version but that's not really  useful.